examples/contour/01-contour-config.yaml

```diff
@@ -68,7 +68,7 @@ data:
     # accesslog-format-string: "...\n"
     # To enable JSON logging in Envoy
     # accesslog-format: json
-    # accesslog-level: info
+    accesslog-level: critical
```

Comment:
probably don't want to change this in the example config

Reply:
sorry just for testing I enabled it, agreed